test/tools/IsolTestOptions.cpp

```diff
@@ -76,6 +76,8 @@ bool IsolTestOptions::parse(int _argc, char const* const* _argv)
 		return false;
 	}
 	enforceViaYul = true;
+	enforceGasTest = (evmVersion() == langutil::EVMVersion::istanbul());
+	enforceGasTestMinValue = 100000;
```

Comment:
Does this code mean that we're just ignoring the command-line parameters for some options in `isoltest` and just hard-code them to defaults? Why?

If there is a legit reason for this then I still think that just failing and telling the user that his values will be ignored might be better.

Reply:
The thing is that I wanted to enforce gas updates in isoltest, same like for enforcing via yul.
If you think we shouldn't run it, I will disable it...